pjlib/include/pj/config_site_test_video.h

```diff
@@ -0,0 +1,5 @@
+#define PJMEDIA_HAS_VIDEO 1
+#define PJMEDIA_SRTP_HAS_DTLS 1
+#define PJMEDIA_HAS_G7221_CODEC 1
+#define PJ_EXCLUDE_BENCHMARK_TESTS 1
+#define PJ_CLI_MAX_CHOICE_VAL 64
```

Comment:
perhaps we can replace this file with config_site_test_full.h+PJMEDIA_HAS_VIDEO?